db/migrate/20160109210511_add_first_move_to_pieces.rb

```diff
@@ -0,0 +1,5 @@
+class AddFirstMoveToPieces < ActiveRecord::Migration
+  def change
+    add_column :pieces, :first_move, :integer
```

Comment:
I'd change this to a `boolean` instead of an enum. http://api.rubyonrails.org/classes/ActiveRecord/Migration.html